include/wx/html/htmlcell.h

```diff
@@ -115,8 +115,15 @@ class WXDLLIMPEXP_HTML wxHtmlRenderingStyle
 class WXDLLIMPEXP_HTML wxDefaultHtmlRenderingStyle : public wxHtmlRenderingStyle
 {
 public:
+    explicit wxDefaultHtmlRenderingStyle(wxWindowBase const* wnd = 0)
```

Comment:
We use west const in wx and also use `NULL`, so:
```suggestion
    explicit wxDefaultHtmlRenderingStyle(const wxWindowBase* wnd = NULL)
```